services/registry/docker-compose.aws.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.7'
+services:
+  registry:
+      image: registry:2
```

Comment:
there is no need to repeat image here since ``docker-compose.aws.yml`` is a partial config and the image is defined in the main ``docker-compose.yml``.